hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleExecutionSpecs.java

```diff
@@ -1571,11 +1598,11 @@ public HapiSpec executionWithDefaultPayerButNoFundsFails() {
                             Assertions.assertEquals(
                                     INSUFFICIENT_PAYER_BALANCE,
                                     triggeredTx.getResponseRecord().getReceipt().getStatus(),
-                                    SCHEDULED_TRANSACTION_SHOULD_NOT_BE_SUCCESSFUL);
+                                    SCHEDULED_TRANSACTION_MUST_NOT_SUCCEED);
```

Comment:
Is there a difference between the mono and mod code that makes these changes necessary? (There are quite a few  of them in this file) 
And, does mono service also pass with these changes?

Reply:
1. Yes, there are differences in Mono, lots of weirdness around execution
2. Yes, Mono service passes with these changes
3. (not asked) There are some additional changes needed, mostly because the tests in this class are extremely order-dependent and fail in HapiTest because the order is random (or at least not exactly what the old suite expects).  Whitelist management is a big issue, and some of these tests unintentionally assume a prior test "set up" some balances (for instance).  The next PR, which focuses on execution tests, will make quite a few changes to make these tests reliable and not order-dependent.